src/library/scala/collection/convert/DecorateAsJava.scala

```diff
@@ -53,10 +53,12 @@ import scala.language.implicitConversions
  *  - `scala.collection.Set` => `java.util.Set`
  *  - `scala.collection.Map` => `java.util.Map`
  *
- *  @author Martin Odersky
+ *  The following one way conversion is provided via `asScala`:
```

Comment:
`DecorateAsJava` does not contain this but it doesn't contain any `asScala` methods even though line 20 notes that it does. More evidence this comment should be deleted.